content/vault/v1.20.x/content/docs/deploy/kubernetes/vso/csi/index.mdx

```diff
@@ -0,0 +1,57 @@
+---
+layout: docs
+page_title: Direct secrets
+description: >-
+  Mount secrets from Vault directly into application containers with the Vault
+  Secrets Operator CSI driver.
+---
+
+# Direct secrets
+
+@include 'alerts/enterprise-only.mdx'
+
+While the Vault Secrets Operator lets you store Vault secrets in clusters as native
+Kubernetes Secrets, you can also use the "direct secrets" feature to mount Vault secrets directly to containers as [volumes](https://kubernetes.io/docs/concepts/storage/volumes/).
```

Comment:
nit: can we call these "ephemeral volumes" throughout. They are defined as such in the `CSIDriver` resource that ships with the chart.